src/08_comprehensions.py

```diff
@@ -12,13 +12,19 @@
 
 y = []
```

Comment:
This will range from 0 to 10 not 1-5 try a list comprehension instead
`y = [x for x in range(6)]`